src/ConfigParser.h

```diff
@@ -129,13 +129,6 @@ class ConfigParser
      */
     static bool NextKvPair(char * &key, char * &value);
 
-    // TODO: Convert into a non-static method after exposing the current parser.
-    /// Extract, validate, and store the ACL key parameter for ACL types
-    /// declared using "acl aclname type key argument..." declaration that
-    /// require unique key values for each aclname+type combination.
-    /// Key comparison is case-insensitive.
-    static void SetAclKey(SBuf &keyStorage, const char *keyParameterName);
```

Comment:
This function was moved to Acl::SetKey() without implementation changes, apart from replacing its internal ConfigParser::strtokFile() call with an external parameter. We need this to deal with optional ACL attributes in ACLCertificateData, where an attribute must be extracted and analysed whether it is a genuine ACL key before subjecting it to Acl::SetKey() checks.